command/root.go

```diff
@@ -62,6 +62,7 @@ func ExecuteContext(ctx context.Context) error {
 	rootCmd.PersistentFlags().StringVarP(&opts.StartSignal, "start-signal", "s", "", "wait for start signal")
 	rootCmd.PersistentFlags().BoolVar(&opts.InsecureTLS, "insecure", false, "disable tls verification")
 	rootCmd.PersistentFlags().IntVar(&opts.RateLimit, "rate-limit", 500*1024, "bytes per second rate limit for UDP output")
+	rootCmd.PersistentFlags().IntVar(&opts.LogReaderBuffer, "log-reader-buf", 500*1024, "buffer size in bytes of the log reader")
```

Comment:
This is the maximum size of buffer that may be allocated during log scanning, correct?

Is this needed to allow scanning lines longer than the default `MaxScanTokenSize = 64 * 1024`?

Reply:
Correct, until we have support for multiline I encountered cases where this is needed. Maybe defaulting to 64*1024 is enough though.

Reply:
I'm fine with making it bigger. Can you change the description to indicate this is the max.